go/apps/ingress/services/router/service.go

```diff
@@ -0,0 +1,162 @@
+package router
+
+import (
+	"context"
+
+	internalCaches "github.com/unkeyed/unkey/go/internal/services/caches"
+	"github.com/unkeyed/unkey/go/pkg/cache"
+	"github.com/unkeyed/unkey/go/pkg/codes"
+	"github.com/unkeyed/unkey/go/pkg/db"
+	"github.com/unkeyed/unkey/go/pkg/fault"
+	"github.com/unkeyed/unkey/go/pkg/otel/logging"
+)
+
+// regionProximity maps AWS regions to their closest regions in order of proximity.
+var regionProximity = map[string][]string{
```

Comment:
later this would be ping based instead in some table so we know whats closests and healthy.